pkg/asset/rhcos/image.go

```diff
@@ -163,6 +164,17 @@ func osImage(config *types.InstallConfig) (string, error) {
 		return osimage, nil
 	case none.Name:
 		return "", nil
+	case nutanix.Name:
+		op := config.Platform.Nutanix
+		if op != nil {
+			if oi := op.ClusterOSImage; oi != "" {
+				return oi, nil
+			}
+		}
+		if a, ok := streamArch.Artifacts["nutanix"]; ok {
+			return rhcos.FindArtifactURL(a)
+		}
+		return "", fmt.Errorf("%s: No Nutanix build found", st.FormatPrefix(archName))
```

Comment:
The name of the artifact in the stream is "nutanix".
```suggestion
		return "", fmt.Errorf("%s: No nutanix build found", st.FormatPrefix(archName))
```